.eslintrc

```diff
@@ -69,6 +69,25 @@
         "browser": false,
         "node": true
       }
+    },
+    {
+      "files": ["packages/components/src/**/*.stories.{ts,tsx}"],
+      "rules": {
+        "no-restricted-imports": [
+          "error", {
+            "patterns": [
+              {
+                "group": [".storybook"],
+                "message": "Do not import files from `.storybook/` into `src`. If needed, move files into `src/storyUtils` instead."
+              },
+              {
+                "group": ["**/docs/*"],
```

Comment:
To prevent false positives from something like `docs-editor/`. Ideally this could be an even more specific path, but was tough to catch relative imports then